src/main/java/vampireNumbers/VampireNumbersMain.java

```diff
@@ -0,0 +1,7 @@
+package vampireNumbers;
+
+public class VampireNumbersMain {
+    public static void main(String[] args) {
+
+    }
+}
```

Comment:
this main method does nothing. Why do you need it?

Reply:
This method was added to Calculation fixed commit accidentally. Now VampireNumbersMain.java is fully implemented.